packages/amplication-client/src/Resource/break-the-monolith/BtmButton.tsx

```diff
@@ -93,7 +93,7 @@ export const BtmButton: React.FC<Props> = ({
   const [showConfirmation, setShowConfirmation] = useState(false);
 
   const [tooltipTextStart, setTooltipTextStart] = useState<string>("");
-  const [tooltipTextEnd, setTooltipTextEnd] = useState<string>("");
+  const [tooltipTextEnd, setTooltipTextEnd] = useState<string>(defaultTextEnd);
```

Comment:
@mulygottlieb, please remove the init value; I did a fix for the correct value that needs to be sent for each license type